schemas/layer-1.cue

```diff
@@ -118,11 +118,17 @@ package schemas
 
 #Mapping: {
 	"reference-id": string @go(ReferenceId)
-	identifiers: [...string]
+	entries: [...#MappingEntry]
 	// Adding context about this particular mapping and why it was mapped.
 	remarks?: string
 }
 
+#MappingEntry: {
+	"reference-id": string @go(ReferenceId)
+	strength: string
```

Comment:
What is this? Can we enumerate its values in a closed set?

Reply:
Strength is the focus of the issue this resolves, #86. It was erroneously a string instead of int, but that's been corrected in a recent commit.